LambdatestEcom/Tests/CartTests.cs

```diff
@@ -1,12 +1,69 @@
-﻿using System;
+﻿using LambdatestEcom.Pages;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using DecoratorDesignPatternTests.Models;
+using LambdatestEcom.Pages;
+using Microsoft.Playwright;
+using System.Xml.Linq;
+
 
 namespace LambdatestEcom.Tests
 {
-    internal class CartTests
+    [Parallelizable(ParallelScope.Self)]
+    [TestFixture]
+    internal class CartTests : UITestFixture
     {
+        [Test]
+        public async Task SearchingProductAndShoppingCartTest()
+        {
+            // Arrange
+            DateTime now = DateTime.Now;
+            string randomString = now.ToString("yyyyMMddHHmmss");
+
+            var homePage = new HomePage(page);
+            var catalogPage = new CatalogPage(page);
+            var productPage = new ProductPage(page);
+            var shopCartPage = new ShopCartPage(page);
+            var addedProductRow = page.Locator("#content")
```

Comment:
locators must not be in the test. You need one function in ShopCartPage, like GetAddedProductLocator(string productName)
so you can use it like 
var addedProductRow = GetAddedProductLocator("HP LP3065");
var addedProductRow2 = GetAddedProductLocator("iPod Classic");